tests/files/example-source-pipepiline-run.yaml

```diff
@@ -13,7 +13,18 @@ spec:
         {}
   workspaces:
     - name: ws-context-dir
-      emptyDir: {}
+      volumeClaimTemplate:
+        metadata:
+          name: source-containter-build-pvc
```

Comment:
binary-container-context-pvc

Reply:
I renamed the build to context, and fixed the typo. But leaving the source part, as the name suggests